### PR TITLE
[TASK] Add export query event

### PR DESCRIPTION
**Pull request by torben-fr**
An event for manipulating the export query is added equivalently to the already existing event for the check query

### CODE REVIEW

File: Classes/Export/Event/AlternateExportQueryEvent.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+     *
+     * @param string $exportKey
+     * @return bool
+     */
+    public function checkExportConfigExists(string $exportKey): bool
+    {
+        return $exportKey === $this->exportKey;
+    }
+
+    public function alternateExportQuery(string $export)
```
Comment (calien666) [inline Classes/Export/Event/AlternateExportQueryEvent.php]:
```suggestion
    public function alternateExportQuery(string $export): void
```

File: Classes/Export/Event/AlternateExportQueryEvent.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+namespace Calien\Xlsexport\Export\Event;
+
+use Psr\EventDispatcher\StoppableEventInterface;
+
+final class AlternateExportQueryEvent implements StoppableEventInterface
+{
+    /**
+     * @var string
+     */
```
Comment (calien666) [inline Classes/Export/Event/AlternateExportQueryEvent.php]: superfluous DocBlock, please remove

File: Classes/Export/Event/AlternateExportQueryEvent.php [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    protected string $exportKey = '';
+
+    /**
+     * @var array
+     */
```
Comment (calien666) [inline Classes/Export/Event/AlternateExportQueryEvent.php]: Please describe the array a bit more, as we try to add phpstan at a later point.
```suggestion
    /**
     * @var array{
     *     export: string,
     *     manipulated?: bool
     *  }
     */
```

File: Classes/Export/Event/AlternateExportQueryEvent.php (hunk 1 above)
Comment (torben-fr) [reply Classes/Export/Event/AlternateExportQueryEvent.php]: Added in 15e899cad26a2df606b5af0bff2cf8764146a5ab

File: Classes/Export/Event/AlternateExportQueryEvent.php (hunk 2 above)
Comment (torben-fr) [reply Classes/Export/Event/AlternateExportQueryEvent.php]: Removed in 15e899cad26a2df606b5af0bff2cf8764146a5ab

File: Classes/Export/Event/AlternateExportQueryEvent.php (hunk 3 above)
Comment (torben-fr) [reply Classes/Export/Event/AlternateExportQueryEvent.php]: More detail in 15e899cad26a2df606b5af0bff2cf8764146a5ab

**Pull request merged**